docs/tutorials/ifci_log_qgroundcontrol.rst

```diff
@@ -0,0 +1,106 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _ifci_log:
+
+**************************************************************
+Exporting Logs from PX4 Flight Controller using QGroundControl
+**************************************************************
+
+.. note::
+    This tutorial is only applicable for PX4 firmware.
+
+
+Exporting Logs 
+==============
+
+Motor telemetry data can be found in the flight controller logs if you are using PX4 firmware with IFCI enabled. Please refer to the documentation on :ref:`Setting up PX4 Firmware for Use with IFCI <ifci_px4_flight_controller>` to make sure your flight controller is configured properly. 
```

Comment:
@Vertiq-Ben You can get telemetry data even if you aren't using IFCI. DroneCAN has all sorts of telemetry that can get saved in a log. Is this a general page about getting logs from a flight controller, or one focused only on IFCI logs? DroneCAN and DSHOT can give you telemetry just as easily that gets logged. We should either split out the IFCI stuff into its own subsection of this page that says "here's specifics about using our fun IFCI stuff" and have the rest of the page be more general, or specify that this is only about IFCI logs. Though I favor the former option. The stuff you have about downloading logs and the existence of plot juggler is generally true, its just intermingled with very specific IFCI stuff